crypto/x509/v3_addr.c

```diff
@@ -343,8 +345,13 @@ static int range_should_be_prefix(const unsigned char *min,
     unsigned char mask;
     int i, j;
 
-    if (memcmp(min, max, length) <= 0)
-        return -1;
+    /*
+     * It is the responsibility of the caller to confirm min < max. We don't use
+     * ossl_assert() here since we have no way of signalling an error from this
+     * function - so we just use a plain assert instead.
+     */
+    assert(memcmp(min, max, length) <= 0);
```

Comment:
The comment doesn't match the actual check. The comment should say `min <= max`.

Reply:
Fixed.